src/components/MovieBanner.tsx

```diff
@@ -3,21 +3,17 @@ import { useEffect, useState } from 'react';
 import Container from './Container';
 import PopularMovie from './PopularMovie';
 import { Movie } from '../models/index';
+import api from 'src/service/api';
```

Comment:
separate line with api service

Reply:
Use relative import, as done in other places

Reply:
changed to relative imports. I basically use a plugin in VS Code that allows doing auto import, hence I didn't pay attention at once